templates/login.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    {% block content %}
+    <h1>Login</h1>
+    <form method="POST" action="{{ url_for('login') }}">
```

Comment:
Атрибут `action` не нужен, так как этот шаблон контролируется функцией представления.
```suggestion
    <form method="POST">
```